inlang/documentation/reference/tableOfContents.md

```diff
@@ -0,0 +1,37 @@
+# Table of Contents
+
+inlang's website is using a custom structure for the table of contents. A real example of the structure can be found here: [tableOfContents.json](https://github.com/inlang/monorepo/blob/main/inlang/documentation/tableOfContents.json)
+
+## Structure
+
+### Variant 1
+
+This variant can be used for a flat table of contents.
+
+```json
+[
+	{
+		"path": "path-to-file.md",
+		"slug": "slug-of-url", // has to be unique
+		"title": "Title of the page",
+		"description": "Description of the page",
+	}
+]
+```
+
```

Comment:
@floriandwt 

left over from the table of contents on marketplace?

this seems to be redundant. we don't need to expose internal stuff on the docuementaiton site.